### PR TITLE
fix/changelog

### PR DESCRIPTION
**Pull request by yehor-podporinov**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-## [Unrealised]
+## [Unreleased]
+#### Added
+- Modules trees for unverified users
+- `UnverifiedUserModuleItem` component
+
+#### Fixed
+- `ModuleTreesItem` component
+
+## [0.4.0] - 2023-07-05
```
Comment (ArtSuslov) [inline CHANGELOG.md]: - how did you add tag to changelog and unreleased at the same time?
- you removed brackets on other tags and added brackets to this one

File: package.json [hunk 2]
```diff
@@ -1,6 +1,5 @@
 {
   "name": "access-control-system",
-  "version": "0.3.0",
```
Comment (ArtSuslov) [inline package.json]: ?

File: CHANGELOG.md [hunk 3]
```diff
@@ -73,7 +81,6 @@ for further information about branching and tagging conventions.
 #### Under the hood changes
 - Initiated and setup project
 
-[Unreleased]: https://gitlab.com/distributed_lab/acs/acs-admin-panel/compare/0.3.0...main
-[0.3.0]: https://gitlab.com/distributed_lab/acs/acs-admin-panel/compare/0.2.0...0.3.0
-[0.2.0]: https://gitlab.com/distributed_lab/acs/acs-admin-panel/compare/0.1.0...0.2.0
-[0.1.0]: https://gitlab.com/distributed_lab/acs/acs-admin-panel/tags/0.1.0
+[Unreleased]: https://github.com/acs-dl/admin-panel/compare/v0.4.0...main
+[0.4.0]: https://github.com/acs-dl/admin-panel/compare/v0.3.0...v0.4.0
+[0.3.0]: https://github.com/acs-dl/admin-panel/tree/v0.3.0
```
Comment (ArtSuslov) [inline CHANGELOG.md]: why did you remove prev tags?

**Review by ArtSuslov — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (yehor-podporinov) [reply package.json]: done

File: CHANGELOG.md (hunk 3 above)
Comment (yehor-podporinov) [reply CHANGELOG.md]: discussed, done

**Review by ArtSuslov — APPROVED** [review]

**Pull request merged**